Dockerfile

```diff
@@ -255,7 +255,7 @@ RUN patchelf --add-needed libm-2.23.so /opt/deps/libvarnishapi.so.1.0.4
 ###### Python Plugin Image ######
 FROM collectd as python-plugins
 
-RUN python3 -m pip install yq &&\
+RUN python3 -m pip install --upgrade pip && python3 -m pip install yq &&\
```

Comment:
It may be best to pin the pip version, e.g. `pip install --upgrade pip==21.0.1`, to avoid any issues that could be introduced with future pip versions.

Reply:
Good idea. I added the suggested version as a Dockerfile ARG, alongside a similar one for golang at the top of the file.